api/private/cerbos/runtime/v1/runtime.proto

```diff
@@ -135,6 +137,15 @@ message Expr {
   google.api.expr.v1alpha1.CheckedExpr checked = 2;
 }
 
+message Output {
+  message On {
+    Expr condition_not_met = 1;
+  }
+
+  Expr expr = 1;
+  On on = 2;
+}
+
```

Comment:
```suggestion
message Output {
  message On {
    Expr condition_met = 1;
    Expr condition_not_met = 2;
  }

  On on = 1;
}

```

Ideally this is how we'd have represented it in the policy if it wasn't for the backward compatibility problem.